IntrepidSwiftWisdom.podspec

```diff
@@ -1,22 +0,0 @@
-Pod::Spec.new do |s|
-  s.name          = "IntrepidSwiftWisdom"
```

Comment:
It looks like a few projects use IntrepidSwiftWisdom. Is it ok to remove this spec?